changelog/638.feature.rst

```diff
@@ -0,0 +1,3 @@
+Add :func:`on_raw_member_remove` and :func:`on_raw_member_update` events.
+
+Along with these new events are the :class:`RawGuildMemberRemoveEvent` and :class:`RawGuildMemberUpdateEvent` models.
```

Comment:
```suggestion
Add :func:`on_raw_member_remove` and :func:`on_raw_member_update` events, with their respective :class:`RawGuildMemberRemoveEvent` and :class:`RawGuildMemberUpdateEvent` models.
```
Multiline changelog entries will be rendered in a bit of a weird way, it's generally better to have things on one line or use bullet points for the following lines.

Reply:
9e8ad04